ScoutSuite/providers/gcp/rules/findings/cloudstorage-bucket-no-logging.json

```
@@ -2,9 +2,9 @@
     "dashboard_name": "Buckets",
     "description": "Buckets should have logging enabled",
     "rationale": "By enabling access and storage logs on target Storage buckets, it is possible to capture all events which may affect objects within target buckets (CIS 5.3).",
-    "path": "cloudstorage.buckets.id",
+    "path": "cloudstorage.projects.id.buckets.id",
     "conditions": [ "and",
-        [ "cloudstorage.buckets.id.logging_enabled", "false", "" ]
+        [ "cloudstorage.projects.id.buckets.id.logging_enabled", "false", "" ]
```

Seems like there might be false negatives here. If get_logging fails, this value will be `None`, and the bucket should be flagged.

```suggestion
        [ "cloudstorage.projects.id.buckets.id.logging_enabled", "false", "" ],
        [ "cloudstorage.projects.id.buckets.id.logging_enabled", "equal", "None" ],

```